index.js

```diff
@@ -42,6 +43,7 @@ const FaucetSmartContractAddress = '6dba10422e368813802877a85039d3985d96760ed844
 const StorageSmartContractAddress = '6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7';
 const MinerSmartContractAddress = '6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d9';
 const InterestPoolSmartContractAddress = 'cf8d0df9bd8cc637a4ff4e792ffe3686da6220c45f0e1103baa609f3f1751ef4';
+const cachedNonceTime = 5*60*1000       //5 Minutes into milli seconds
```

Comment:
A more descriptive variable name would be something like:

```suggestion
const cachedNonceResetInterval = 5*60*1000       //5 Minutes into milli seconds
```